jms/1.7/modules/ROOT/pages/jms-connector-reference.adoc

```diff
@@ -489,9 +489,9 @@ JMS Subscriber for Destinations, allows to listen for incoming Messages
 [%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Properties a| <<JMSMessageProperties>> |  |  |
-| Headers a| Any |  |  | x
-| Ack Id a| String |  |  |
+| Properties a| <<JMSMessageProperties>> | Container element for all the properties present in a JMS Message. |  |
+| Headers a| Any | All the possible headers of a JMS message. |  | x
+| Ack Id a| String | The session Ack ID required to acknowledge the current message if one is available, otherwise, is null.  |  |
```

Comment:
```suggestion
| Ack Id a| String | The session Ack ID required to acknowledge the current message if one is available, otherwise, it is null.  |  |
```